app/routes/games.locks-challenge.rules.tsx

```diff
@@ -0,0 +1,15 @@
+export default function GamesLocksChallengeStreamingRules() {
+    return (
+      <>
+        <h2>Locks Challenge Rules</h2>
+        <p>
+          Start the season with 0 points. Each week, you can select as many teams to win

```

Comment:
Going to try a rewrite of the rules for clarity: (how are ties broken?)

Each week, you get to pick the winner of as many games as you'd like to choose. At the end of the week, if none of the teams you selected lost, you get one point per team you selected that won.

However, if any of your teams lose in that week, you score zero points for that week.

Ties don't count as a loss, but you don't get any points for a win from them either.

The player that scores the most points over the course of the season wins. There are no penalties for not picking any teams in a week besides foregoing the opportunity to earn points that week.

Reply:
lgtm, I was probably going to make them using chatgpt before the season starts. Could potentially break end of season ties by looking at which player had the most correct picks.

Reply:
That seems fine to me. Are we handling that in the standings sorting currently? If not, we can add that in separately.

Reply:
not currently being handled but I will add that logic in before the season starts